tests/suites/test_suite_ssl.function

```diff
@@ -1537,6 +1537,19 @@ exit:
     return ret;
 }
 
+/*
+ * Perform data exchanging between \p ssl_1 and \p ssl_2. Both of endpoints
+ * must be initialized and connected beforehand.
+ *
+ * \retval  0 on success, otherwise error code.
+ */
+int exchange_some_data( mbedtls_ssl_context *ssl_1,
```

Comment:
I don't really like the `some` part of it, would rather recommend naming it "exchange_data", with the amount being a parameter, a hard-coded value in a particular test.